packages/blockchain-wallet-v4-frontend/src/scenes/Refer/Sofi/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Redirect, RouteComponentProps } from 'react-router-dom'
+
+const SofiReferralComponent: React.FC<RouteComponentProps> = ({ location }) => {
+  const searchParams = new URLSearchParams(location.search)
+  const hasAllRequiredJwtParams = ['aesCiphertext', 'aesIV', 'aesTag', 'aesKeyCiphertext'].every(
```

Comment:
Possibly nothing but could parameters change? If so, we could make them a configuration instead so as to not have to deploy the entire app to swap them.

Reply:
good point. these would not change, they are the ones required for the sofi migration. but also, the spec has been changed and I have a new commit I am waiting to push up that has this bit entirely removed 🙈